orchestra/project.py

```diff
@@ -1,10 +1,17 @@
+import threading
 from orchestra.communication.slack import create_project_slack_group
 from orchestra.google_apps.convenience import create_project_google_folder
 from orchestra.models import Project
 from orchestra.models import WorkflowVersion
 from orchestra.utils.task_lifecycle import create_subsequent_tasks
 
 
+def _run_create_project_google_folder_in_background(project):
+    t = threading.Thread(target=create_project_google_folder, args=(project,))
+    t.daemon = True
```

Comment:
curiously, what's this for?

Reply:
I misunderstood the docs. Now I see that `daemon = True` is not what we want. Removed.